pkgs/tools/misc/android-tools/default.nix

```diff
@@ -33,6 +42,12 @@ stdenv.mkDerivation rec {
     export GOCACHE=$TMPDIR/go-cache
   '';
 
+  postInstall = ''
+    install -Dm755 ../vendor/avb/aftltool.py -t $out/bin
+    install -Dm755 ../vendor/avb/avbtool.py -t $out/bin
+    install -Dm755 ../vendor/mkbootimg/mkbootimg.py $out/bin/mkbootimg
```

Comment:
Is there any reason to only remove the `.py` suffix from `mkbootimg` (I've never used those tools).

Reply:
`aftltool` imports `avbtool` so we need to keep .py suffix. `aftltool` is not a common tool I don't know we should include it or not.

Reply:
There is a [patch](https://build.opensuse.org/package/view_file/hardware/android-tools/fix-install-python-tools.patch?expand=1) that applied on openSUSE package which also installs these tools and keeps `.py` suffix but not sure OBS is reproducible.

Reply:
Ok, should be fine then, thanks!